### PR TITLE
Feature/product page

### PR DESCRIPTION
**Pull request by Diegobr21**
Product Page now has/does:
*Product Card
*Random reviews generated from ReviewService.cs
*EditForm to create reviews (not saved just UI with DataAnnotations)

### CODE REVIEW

File: Elecritic/Pages/ProductPage.razor.cs [hunk 1]
```diff
@@ -21,6 +22,16 @@ public partial class ProductPage {
             //reviewService.SaveReview(review);
             review.ClearReview();
         }
+
+        //ReviewService
+        [Inject]
+        public ReviewService reviewService { get; set; }
```
Comment (netotz) [inline Elecritic/Pages/ProductPage.razor.cs]: rename property to match naming conventions to `ReviewService`

File: Elecritic/Pages/ProductPage.razor.cs [hunk 2]
```diff
@@ -1,14 +1,67 @@
...
-using Elecritic.Models;
 using Microsoft.AspNetCore.Components;
 
 namespace Elecritic.Pages {
     public partial class ProductPage {
 
         [Parameter]
         public string ProductId { get; set; }
+
+        public ReviewModel review { get; set; }
```
Comment (netotz) [inline Elecritic/Pages/ProductPage.razor.cs]: rename property to match naming conventions to `ReviewModel`

**Review by netotz — CHANGES_REQUESTED** [review]
In addition to the comments below, I suggest the following changes:

- Add docstrings to classes, properties and methods to explain why they are there and their purpose
- Remove unnecessary imports and clean up code
- Remove `@code { }` blocks from .razor files

**Review by netotz — APPROVED** [review]

**Review by netotz — APPROVED** [review]
nice!

**Comment by Diegobr21** [comment]
> nice!

Thanks  senpai

**Pull request merged**